packages/next/src/pages/_error.tsx

```diff
@@ -88,23 +88,23 @@ export default class Error<P = {}> extends React.Component<P & ErrorProps> {
               /* CSS minified from
                 body { margin: 0; color: #000; background: #fff; }
                 .next-error-h1 {
-                  border-right: 1px solid rgba(0, 0, 0, .3);
+                  border-right: 1px solid rgb(0, 0, 0, .3);
```

Comment:
If we're going for modern syntax...

```suggestion
                  border-right: 1px solid rgb(0 0 0 / .3);
```

Reply:
@wojtekmaj 

Thank you for your suggestion.

I understand that space-separated is modern.

However, since Comma-separated is used in many parts of this framework, I would like to keep rgba to rgb only in this PR.